spec/istore/istore_io_spec.rb

```diff
@@ -77,23 +77,22 @@
           expect{query("SELECT '1=>2,2='::#{type}")}.to throw_error "invalid input syntax for istore: \"1=>2,2=\""
         end
       end
-
       describe 'array input' do
         it 'should parse normal arrays tuple' do
           query("SELECT '([1,2], [11, 22])'::#{type}").should match \
           '"1"=>"11", "2"=>"22"'
         end
-        it 'should parse uneven arrays tuple' do
-          query("SELECT '([1,2,2], [11, 22])'::#{type}").should match \
```

Comment:
I would prefer to keep the test just change the expectation to raise

Reply:
yes, I did that in latest commits